e2e/key/key.go

```diff
@@ -315,6 +316,156 @@ func (c *ctx) singularityResetKeyring(t *testing.T) {
 	}
 }
 
+func (c *ctx) singularityKeyPush(t *testing.T) {
+	tests := []struct {
+		name          string
+		cmdArgs       []string
+		expectedExit  int
+		expectedRegex string
+	}{
+		{
+			name:          "help",
```

Comment:
Just to make it a little more clear

```suggestion
￼			name:          "push help",
```